samples/sample-production/app/src/main/java/co/touchlab/kermitSample/FirstFragment.kt

```diff
@@ -8,12 +8,12 @@
  * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
  */
 
-package co.touchlab.KermitSample
+package co.touchlab.kermitSample
```

Comment:
package names should be all lower case

Reply:
https://kotlinlang.org/docs/coding-conventions.html#naming-rules

> you do need to use multiple words, you can either just concatenate them together or use the camel case (org.example.myProject).

Also, IDE doesn't complain anymore with `kermitSample` but it was complaining with `KermitSample`.

Let me know if you think, as a TL practice, we should have package name entirely `lower case`. I'll make the change then.